heaps/min_heap.py

```diff
@@ -19,19 +19,38 @@ def __init__(self):
     def add(self, key, value = None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
-        pass
+        if value == None:
+            value = key
+        new_node = HeapNode(key,value)
+
+        # add new_node to end of heap
+        self.store.append(new_node)
+        self.heap_up(len(self.store) - 1)
+
 
     def remove(self):
```

Comment:
✨Nice! Same thing on space complexity as in `add` because of your call to `heap_down` though

Really like your comments!